### PR TITLE
Product Page : Remove v1 & Set v2 as main for Sanity Tests

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Product Page : Remove v1 & Set v2 as main for Sanity Tests
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI is :green_circle: & Nightly is :green_circle: : https://github.com/Progi1984/ga.tests.ui.pr/actions/runs/5528915438
| Fixed ticket?     | N/A
| Related PRs       | N/A
| Sponsor company   | @PrestaShopCorp

### CODE REVIEW

File: tests/UI/pages/BO/catalog/productsV2/add/index.ts [hunk 1]
```diff
@@ -174,7 +174,7 @@ class CreateProduct extends BOBasePage {
    * @returns {Promise<string>}
```
Comment (boubkerbribri) [inline tests/UI/pages/BO/catalog/productsV2/add/index.ts:174]: should we also move pages from productV2 to product ?

**Review by boubkerbribri — COMMENTED** [review]
Just one question

File: tests/UI/pages/BO/catalog/productsV2/add/index.ts (hunk 1 above)
Comment (Progi1984) [reply tests/UI/pages/BO/catalog/productsV2/add/index.ts:174]: I will do that after upgrade of functional part. It's just sanity.

**Review by jolelievre — DISMISSED** [review]
Thanks @Progi1984

**Review by nesrineabdmouleh — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by jolelievre** [comment]
Thanks @Progi1984

**Pull request merged**